zbus/tests/basic.rs

```diff
@@ -118,13 +118,20 @@ async fn fdpass_systemd_async() {
     f.metadata().unwrap();
 }
 
-#[cfg(target_os = "linux")]
-#[cfg(feature = "ibus")]
+#[cfg(all(target_os = "linux", feature = "ibus"))]
```

Comment:
For example, this is just a fixup of the commit that adds the `test_ibus`. Also, if you move the commit that adds the testcase after the blocking version addition, you can just squashed this whole ("zb: update the test code of IBus") commit into that one.

Reply:
Ha, I guess it is good to record the code improvements.

Reply:
True but when people look at your commits, they'll only be interested in what changed and why. Also if for any reason (doubtful but possible) we need to revert your changes later, it's easier to do so with atomic (as in, one logical change per commit) commits.

Besides, this goes in